PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+### Checks and guidelines:
+<!-- Mark your checks with 'x' inside the square brackets -->
+
+* [ ] Have you checked that there aren't other open [Pull Requests](../../pulls) for the same update/change?
+* [ ] Have you checked that only **single quotes** are used in the code, apart from the doc-strings?
+* [ ] Have you run a lint program on your code prior to submission?
+* [ ] Have you checked if `python run_tests.py` returns no errors?
+
+Tests output: [link](https://put-your-link-to-your-output.here) <!-- use 'paste.ubuntu.com', 'hastebin.com' or similar -->
+
+<!-- You can erase any part of this template if not applicable to your Pull Request. -->
+
+### Type of change
+
+* [ ] Bug fix
+* [ ] New feature
```

Comment:
Maybe add an "Improvement" type of change (when it's not a new feature nor a bug fix but still but still a modification)

Reply:
indeed